src/TaskRunner/Commands/CloneCommands.php

```diff
@@ -168,27 +173,28 @@ public function downloadDump(array $options = [
             return $this->collectionBuilder()->addTaskList($tasks);
         }
 
-        // Download the file.
+        // Download the .sha file.
+        $this->generateAsdaWgetInputFile('latest.sh1', $options);
         $this->downloadChecksumFile($options);
         $fileContent = file_get_contents('latest.sh1');
         $filename = trim(explode('  ', $fileContent)[1]);
 
-        // Download the file.
+        // Download the .sql file.
+        $this->generateAsdaWgetInputFile($filename, $options);
         $tasks[] = $this->taskExec('wget')
-            ->option('--http-user', $options['asda-user'])
-            ->option('--http-password', $options['asda-password'])
             ->option('-O', $options['dumpfile'] . '.gz')
-            ->option('-nH', $options['asda-url'] . '/' . $filename)
+            ->option('-i', this::TEMP_INPUTFILE)
```

Comment:
Should be `self::TEMP_INPUTFILE` I guess, since it's only used it it's own class.

Reply:
In fact there is a typo there. I would type. $this->TEMP_INPUTFILE instead. I would reserve :: for static members.